release-tag.sh

```diff
@@ -0,0 +1,7 @@
+version=`python setup.py --version`
+eval "$(ssh-agent -s)" #start the ssh agent
+chmod 400 ./openfisca_bot
+ssh-add ./openfisca_bot
```

Comment:
There is a dependency on this file being there, and thus on Travis having decrypted the file. This should at least be documented at the top of the file.